modules/gapi/src/backends/ie/giebackend.cpp

```diff
@@ -253,10 +265,40 @@ struct IEUnit {
                 }
             }
         }
-#endif
-
+#endif // INF_ENGINE_RELEASE < 2020000000
+        // In dnn, after ordinary extensions they try to add custom <InfEngineExtension> as
+        // "some of networks can work without a library of extra layers.
+        //  OpenCV fallbacks as extensions"
+        // Do WE need this? (dnn/src/op_inf_engine.cpp:824)
```

Comment:
It's a mechanic to wrap OpenCV layers for unsupported IE layers. No need to use in G-API

Reply:
OK, thank you!